external/fv3fit/tests/_regtest_outputs/test_sklearn.test_random_state_uniform_reproducibility.out

```diff
@@ -1,2 +1,2 @@
-1.19.4
+1.19.5
```

Comment:
This says 1.19.5 but the test_numpy_version says 1.19.4 - what's up?

Reply:
Oh, this must be old from before I moved the numpy version test out of this test function. I'll delete.